lib/paper_trail.rb

```diff
@@ -46,7 +48,11 @@ module PaperTrail
   class << self
     # @api private
     def clear_transaction_id
```

Comment:
Should this and other `# @api private` methods, such as `transaction_id=`, have a deprecation warning?  Is your rationale that even if they're private, clients might be using them, and we want to deprecate them in the same way?

Reply:
Going forward, all transaction id methods should be `@api private`. I'll fix that in the depr. warnings and comments.